common/core/desktop/meson.build

```diff
@@ -6,7 +6,7 @@
 #
 
 project('keyboardprocessor', 'cpp', 'c',
-        version: '11.0.0',
+        version: '14.0.17',
```

Comment:
Can we avoid rewriting this file each time there is a version update? Can it be a meson.in -> meson.build or something like that?

Reply:
I found the incantation to cat the VERSION.md file.